index.bs

```diff
@@ -3925,97 +3925,43 @@ Transitioning to Proposed Recommendation</h4>
 			<em class="rfc2119">may</em> have removed features
 			identified in the [=Candidate Recommendation Snapshot=] document as [=at risk=]
 			without republishing the specification as a [=Candidate Recommendation Snapshot=].
-	</ul>
-
-	The [=Team=]:
-
-	<ul>
-		<li>
-			<em class="rfc2119">must</em> announce the publication of a [=Proposed Recommendation=]
-			to the <a href="#AC">Advisory Committee</a>,
-			and <em class="rfc2119">must</em> begin an [=Advisory Committee Review=]
-			on the question of whether the specification is appropriate to [=publish=] as a [=W3C Recommendation=].
 
 		<li>
-			<em class="rfc2119">may</em> approve a [=Proposed Recommendation=]
-			with minimal <a href="#implementation-experience">implementation experience</a>
-			where there is a compelling reason to do so.
-			In such a case, the [=Team=] <em class="rfc2119">must</em> explain the reasons for that decision,
-			and that information <em class=rfc2119>must</em> be included in the [=Call for Review=]
-			proposing advancement to [=W3C Recommendation=].
+			<em class="rfc2119">must</em> identify, in the document, where errata are tracked.
 	</ul>
 
-	Since a [=W3C Recommendation=] <em class="rfc2119">must not</em> include any [=substantive changes=]
-	from the [=Proposed Recommendation=] it is based on,
-	to make any [=substantive change=] to a [=Proposed Recommendation=]
-	the [=Working Group=] <em class="rfc2119">must</em> return the specification to [=Candidate Recommendation=]
-	or [=Working Draft=].
-
-	<p id="expandable-rec">
-	A [=Proposed Recommendation=] may identify itself
-	as intending to <dfn>allow new features</dfn>
-	(<a href="#correction-classes">class 4 changes</a>)
-	after its initial publication as a [=Recommendation=],
-	as described in [[#revised-rec-features]].
-	Such an allowance cannot be added
-	to a [=technical report=] previously published as a [=Recommendation=]
-	that did not allow such changes.
-
-	Possible Next Steps:
-
-	<ul>
-		<li>
-			Return to <a href="#revising-wd">Working Draft</a>
-
-		<li>
-			Return to <a href="#transition-cr">Candidate Recommendation</a>
-
-		<li>
-			<a href="#transition-rec">Recommendation status</a>,
-			(the expected next step).
-
-		<li>
-			<a href="#abandon-draft">Discontinued Draft</a></li>
-	</ul>
-
-	[=Advisory Committee representatives=] <em class="rfc2119">may</em> initiate an [=Advisory Committee Appeal=]
-	of the decision to advance the technical report.
+	Additionally,
+	if the document has previously been published as a [=W3C Recommendation=], the [=Working Group=]
+	<em class=rfc2119>must not</em> include any <a href="#correction-classes">class 4 change</a>
+	to that publication
+	unless it was explicitly marked as [=allowing new features=],
+	and <em class=rfc2119>must not</em> include any such marking
+	if not already present.
+
+	If all the criteria above are fulfilled,
```

Comment:
It would be worth being explicit that the Team's trigger to check if the criteria have been fulfilled is a WG Decision to request advancement.

Reply:
It's a little hard to see in the diff because there are a lot of added/removed lines in between, but if you look at the built version, isn't that already made clear by the first sentence in the section?
https://pr-preview.s3.amazonaws.com/frivoal/w3process/pull/868.html#transition-rec

Reply:
I did see that sentence, but for myself, I don't think it's clear that a WG requesting advancement to REC is synonymous with a WG Decision to request advancement. 

It needs to be unambiguous to Chairs what is required; the alternative is that it could be seen as being okay if a WG has some kind of discussion without a clear assessment of consensus and the Chair then makes a request for advancement on behalf of the WG; that would bypass any WG decision policy, for example.

Reply:
Addressed by 66d34bc30cc6cf14518359573a4761f59d46244d, as discussed in the ProcessCG call.